dev/io.openliberty.reporting.internal/src/io/openliberty/reporting/internal/FixReportingComponent.java

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.reporting.internal;
+
+import static org.osgi.service.condition.Condition.CONDITION_ID;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.propertytypes.SatisfyingConditionTarget;
+
+import com.ibm.websphere.kernel.server.ServerInfoMBean;
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.kernel.feature.FeatureProvisioner;
+import com.ibm.ws.kernel.feature.FixManager;
+import com.ibm.ws.kernel.feature.ServerStartedPhase2;
+
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+
+/**
+ * <p>
+ * The main entry point into CVE Reporting service
+ * </p>
+ */
+@Component(immediate = true, configurationPid = "io.openliberty.reporting.internal.CVEReportingComponent", configurationPolicy = ConfigurationPolicy.OPTIONAL)
```

Comment:
If you remove the `.internal` from the metatype, reminder to move it here as well for the `configurationPid`

Reply:
Done.